modules/manual-configuration-image-registry-ecr.adoc

```diff
@@ -22,17 +34,18 @@ Follow the instructions at link:https://docs.aws.amazon.com/AmazonECR/latest/use
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Under the *Image Integrations* section, select *Amazon Elastic Container Registry*.
-+
-The *Configure image integration* modal box opens.
-. Click *New Integration*.
-. Enter the details for the following fields:
-.. *Integration Name*: The name of the integration.
-.. *Types*: Select *Registry*.
+. Under the *Image Integrations* section, select *Amazon ECR*.
+. Click *New integration*, or click in one of the autogenerated integrations to open it, then click *Edit*.
```

Comment:
Just checking, is "click in one of the autogenerated integrations" intended here? or should it just be "click one of...."?